lib/src/platform/src/push_notification_events_internal.dart

```diff
@@ -77,6 +84,8 @@ class PushNotificationEventsInternal implements PushNotificationEvents {
     }
   }
 
+  /// Used to close internal [StreamController] instances
+  // FIXME: This method is not called anywhere, and it probably should be
```

Comment:
I think you've not finished your thought at the end this comment :wink:

Reply:
My bad, I started investigating it and forgot 🤦  I've updated the comment and added link to https://github.com/ably/ably-flutter/issues/382